external/netstandard/netstandard.depproj

```diff
@@ -28,6 +29,12 @@
     <PackageReference Include="System.Runtime.Serialization.Primitives">
       <Version>4.3.0</Version>
     </PackageReference>
+    <PackageReference Condition="'$(NETStandardVersion)' &gt;= 1.3" Include="System.Reflection.TypeExtensions">
```

Comment:
MSBuild can handle numeric comparisons on integers and decimals. https://msdn.microsoft.com/en-us/library/7szfhaft.aspx